config/default.schema.json

```diff
@@ -301,6 +301,11 @@
 			"default": false,
 			"description": "Enables the Identity Management (ErWIn IDM) as account storage."
 		},
+		"FEATURE_IDENTITY_MANAGEMENT_IS_PRIMARY": {
```

Comment:
Primary for what? I understand it, but I don't know if devs who did not listen to our dailies understand this